conseil-lorre/src/main/scala/tech/cryptonomic/conseil/lorre/LorreAppConfig.scala

```diff
@@ -1,14 +1,15 @@
-package tech.cryptonomic.conseil.config
+package tech.cryptonomic.conseil.lorre
```

Comment:
The `argParser` is built with the wrong name ("lorre"), that will show on the console prompt. it should be renamed "conseil-lorre" for consistency with how the program is run